modules/rosa-hcp-sts-creating-a-break-glass-cred-cli.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-creating-a-break-glass-cred-cli_{context}"]
+= Creating a break glass credential for a {hcp-title} cluster
+
+As a {hcp-title} cluster owner, you can use the break glass credential to create temporary administrative client credentials to access your clusters that are configured with custom OpenID Connect (OIDC) token issuers. Creating a break glass credential generates a new cluster-admin `kubeconfig` file. The `kubeconfig` file contains information about the cluster that the CLI uses to connect a client to the correct cluster and API server. You can use the newly generated `kubeconfig` file to allow access to the {hcp-title} cluster.
+
+.Prerequisites
+
+* You have created a {hcp-title} cluster with external authentication enabled. For more information, see _Creating a {hcp-title} cluster that uses external authentication_.
+* You have created an external authentication provider. For more information, see _Creating an external authentication provider_.
+* You have an account with `cluster admin` permissions.
+
+.Procedure
+
+. Create a break glass credential by using one of the following commands:
+
+** Create a break glass credential by using the interactive mode to interactively specify custom settings:
```

Comment:
Per [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command):
```suggestion
** To create a break glass credential by using the interactive mode to interactively specify custom settings, run the following command:
```